packages/labs/analyzer/src/lib/model.ts

```diff
@@ -438,31 +444,31 @@ export class ClassDeclaration extends Declaration {
    * (excluding any inherited members).
    */
   get fields() {
-    return this._fieldMap.values();
+    return [...this._fieldMap.values(), ...this._staticFieldMap.values()];
   }
 
   /**
    * Returns iterator of the `ClassMethod`s defined on the immediate class
    * (excluding any inherited members).
    */
-  get methods(): IterableIterator<ClassMethod> {
-    return this._methodMap.values();
+  get methods() {
```

Comment:
Does removing this return type cause it to implicitly be `Array<ClassMethod>`? Is that something we care about w.r.t. API stability / flexibility?

Reply:
It does, and I don't think we care that much about stability at this exact point. It's not possible to make a generator getter directly (where we could just yield both values() out; it would need to call out to one), and didn't figure it was worth the trouble to keep this an iterator.

Reply:
Oh, separating them allows keeping that return type I guess. Might as well.